src/e2e/java/teammates/e2e/cases/e2e/AdminSearchPageE2ETest.java

```diff
@@ -231,6 +231,11 @@ private void verifyLinkExpansionButtons(StudentAttributes student, InstructorAtt
 
         searchPage.clickExpandInstructorLinks();
         searchPage.clickCollapseStudentLinks();
+        try {
+            Thread.sleep(500);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
Although this works, this may not be very stable. Is there another way possible here, e.g. waiting for certain element to appear?
If not then at least abstract this out to a method in `AppPage`.
```java
public void waitForCollapseAnimation() {
    ThreadHelper.waitFor(500);
}
```